src/main/java/net/atos/client/brp/model/listPersonenParameters.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.client.brp.model;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import javax.ws.rs.QueryParam;
+
+public class listPersonenParameters {
+
+    @QueryParam("fields")
+    private String fields;
+
+    @QueryParam("burgerservicenummer")
```

Comment:
Even als check: is burgerservicenummer juist? Gezien het over een lijst van burgerservicenummers gaat, zoals je ze zelf ook noemt in de list?

Reply:
Ja burgerservicenummer is zoals het attribuut heet in de OpenAPI spec.